pkg/model/dev.go

```diff
@@ -529,6 +529,14 @@ func (dev *Dev) IsHybridModeEnabled() bool {
 	return dev.Mode == constants.OktetoHybridModeFieldValue
 }
 
+func (dev *Dev) GetMode() string {
```

Comment:
could we set the default value when reading the manifest? I think there is already a `setDefaults` method.
That way we make sure if someone access `dev.Mode` directly it will have the right value and we don't need the `GetMode` method

Reply:
Done!